### DISCUSSION TITLE
[UX Enhancements] Focus on Search input & shortcut to copy the code

### DISCUSSION THREAD

**Discussion by dxps** [Authenticator-Extension/Authenticator#1090] [Ideas]
Hi everyone!

Authenticator is a great extension! I'm using it daily, I recommend it to anyone. :relaxed: 

I'd suggest two small things that would improve de UX:
1. When it is open, the focus should be on the _Search_ input.
    - So that you just start typing to filter the entry you need.
    - Yes, using the `/` key, the focus gets on the input. But why not implicitly, or part of user preference.
3. A shortcut to copy the code.
    - When the entry is found, you can use the arrows to get the focus on it.
    - But you have to use the mouse to click on it to copy the code. 
    - A keyboard shortcut would definitely help.

Thanks in advance for considering these!

**Comment by mymindstorm** [comment]
Re: 1, it should already be doing that if you have enough accounts for it to show the search bar by default. Although, getting the focus on open to work was finicky from what I remember. It is likely different per-platform.

Re: 2 it will copy a code that is selected using the enter key. Adding an alias to this function on space would probably help w/ discoverability.

**Comment by dxps** [comment]
@mymindstorm Thanks for the feedback! :pray: 

Related to the search input, it's shown and typing `/` is fine to get the focus on it and start filtering the entries.

Related to copying the code, `Enter` is the key. :blush:  Thanks for it! Tried and it's working fine.
